dissect/target/plugins/general/network.py

```diff
@@ -0,0 +1,54 @@
+from typing import Any, Iterator, Union
+
+from flow.record.fieldtypes.net import IPAddress
+
+from dissect.target.helpers.record import (
+    MacInterfaceRecord,
+    UnixInterfaceRecord,
+    WindowsInterfaceRecord,
+)
+from dissect.target.plugin import Plugin, export, internal
+
+InterfaceRecord = Union[UnixInterfaceRecord, WindowsInterfaceRecord, MacInterfaceRecord]
+
+
+class NetworkPlugin(Plugin):
+    __namespace__ = "network"
+
+    def check_compatible(self) -> None:
+        pass
+
+    @internal
+    def find_interfaces(self) -> list[InterfaceRecord]:
+        yield from ()
+
+    def _get_record_type(self, field_name: str) -> list[Any]:
+        for record in self.find_interfaces():
+            output = getattr(record, field_name, None)
+            if output is None:
+                continue
```

Comment:
```suggestion
            if (output := getattr(record, field_name, None)) is None:
                continue
```
Not sure if this works but I think it does.